backend/src/main/java/gov/cdc/usds/simplereport/api/Translators.java

```diff
@@ -230,4 +232,29 @@ public static String parseState(String s) {
     }
     return symptomsMap;
   }
+
+  public static PersonName consolidateNameArguments(
+      PersonName name, String firstName, String middleName, String lastName, String suffix) {
+    return consolidateNameArguments(name, firstName, middleName, lastName, suffix, false);
+  }
+
+  public static PersonName consolidateNameArguments(
```

Comment:
Is the thought with this that we will eventually convert the frontend to only make calls with `NameInput`, and then remove the translator?

Reply:
That is my hope, yes! Hence tagging some people who do front-end development so they can decide whether it's worth their time. 😆